### PR TITLE
Small fixes based on PR`s comments

### PR DESCRIPTION
**Pull request by lnikitadobrenkol**
refactoring:
1) add string literals where it should be
2) review quotes, now it has the same style everywhere in the project
3) rework try...catch construction, now it works the right way
4) delete redundant rows
5) rebase some constants and rows for better organization
6) rework initialization of cleanedArray from 2 rows to 1 row
7) add check "isArray" to getDuplicates helper

### CODE REVIEW

File: src/server/controllers/uploadController.js [hunk 1]
```diff
@@ -2,28 +2,28 @@ const multer = require('multer');
...
-    }
-};
+    const fileDestination = {
+        destination: function (req, file, callback) {
+            callback(null, filePath);
+        },
+        filename: function (req, file, callback) {
+            callback(null, 'duplicates_(' + currentFileIndex() + ').json');
+        }
+    };
```
Comment (vitaliipasiuta) [inline src/server/controllers/uploadController.js:14]: why did you add this extra tabs?

File: src/server/controllers/uploadController.js [hunk 2]
```diff
@@ -2,28 +2,28 @@ const multer = require('multer');
...
-    filename: function (req, file, callback) {
-        callback(null, "duplicates_(" + currentFileIndex() + ").json");
-    }
-};
+    const fileDestination = {
+        destination: function (req, file, callback) {
+            callback(null, filePath);
+        },
+        filename: function (req, file, callback) {
+            callback(null, 'duplicates_(' + currentFileIndex() + ').json');
```
Comment (vitaliipasiuta) [inline src/server/controllers/uploadController.js:12]: string literal here

File: src/server/services/duplicatesService.js [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
+    let duplicates = getDuplicates(cleanedInputArr);
 
+    try {
+        duplicates = JSON.stringify(duplicates, null, 2);
+    } catch (error) {
+        throw new Error (`Sorry, i can not stringify your object "duplicates". \n ${error}`)
+    }
 
+    fs.writeFileSync(outputFilePath + fileName, duplicates, 'utf-8');
+};
```
Comment (vitaliipasiuta) [inline src/server/services/duplicatesService.js:33]: void function? maybe return some value if success?

File: src/server/controllers/uploadController.js (hunk 2 above)
Comment (lnikitadobrenkol) [reply src/server/controllers/uploadController.js:12]: here I calling the method and it doesn`t work with string literal

File: src/server/services/duplicatesService.js (hunk 3 above)
Comment (lnikitadobrenkol) [reply src/server/services/duplicatesService.js:33]: I am changing this one to async, so there no reason to add something in this current function

File: src/server/services/helpers/getDuplicates.js [hunk 4]
```diff
@@ -1,26 +1,30 @@
 module.exports = function getDuplicates(cleanedArray) {
     let duplicatesObject = {};
-    duplicatesObject[cleanedArray[0][0]] = cleanedArray[0][1];
+    if (Array.isArray(cleanedArray)) {
```
Comment (vitaliipasiuta) [inline src/server/services/helpers/getDuplicates.js:3]: here you checked that `cleanedArray` is array, well done. but you still don't check that it's the two-dimension array. 
cleanedArray[0] - can be not an array and you did cleanedArray[0][0], also can be tricky place

**Review by vitaliipasiuta — APPROVED** [review]
👍

File: src/server/controllers/uploadController.js (hunk 2 above)
Comment (vitaliipasiuta) [reply src/server/controllers/uploadController.js:12]: it should

File: src/server/services/duplicatesService.js [hunk 5]
```diff
@@ -1,33 +1,33 @@
...
-const outputFilePath = "src/outputData/";
+const inputFilePath = 'src/inputData/';
+const outputFilePath = 'src/outputData/';
 
-const indexFilePath = "./src/dataBase/file-indexData.json";
+const indexFilePath = './src/dataBase/fileIndexData.json';
 
-module.exports = function() {
-    const getFileIndex = require('./helpers/readFileIndex');
+let fileName =``;
```
Comment (fedotov) [inline src/server/services/duplicatesService.js:11]: Need space after =
Also string literal should be '' not ``

File: src/server/services/duplicatesService.js [hunk 6]
```diff
@@ -1,33 +1,33 @@
...
-    cleanedInputArr = cleanedInputArr.map((val) => [val[0], val[1].trim()]);
-
-    const duplicates = getDuplicates(cleanedInputArr);
-    fs.writeFileSync(outputFilePath + fileName, JSON.stringify(duplicates, null, 2), 'utf-8');
-};
-
-
-
-
+    const cleanedInputArr = inputArrFile.map((val) => val.trim().split(':')).map((val) => [val[0], val[1].trim()]);
```
Comment (fedotov) [inline src/server/services/duplicatesService.js:22]: To simplify reading
`inputArrFile`
`    .map((val) => val.trim().split(':'))`
`    .map((val) => [val[0], val[1].trim()]);`

File: src/server/services/helpers/setFileIndex.js [hunk 7]
```diff
@@ -10,7 +10,7 @@ module.exports = function downloadIndex() {
     try {
         indexObject = JSON.parse(indexData);
     } catch (error) {
-        console.error("Sorry, i can`t parse to JSON indexDATA file. \n" + error);
+        throw new Error (`Sorry, i can not parse to JSON indexDATA file. \n ${error}`)
```
Comment (fedotov) [inline src/server/services/helpers/setFileIndex.js:13]: Extra space, semicolon is required
throw new Error(`Sorry, i can not parse to JSON indexDATA file. \n${error}`);

File: src/server/services/helpers/setFileIndex.js [hunk 8]
```diff
@@ -22,7 +22,7 @@ module.exports = function downloadIndex() {
     try {
         indexString = JSON.stringify(indexObject, null, 2);
     } catch (error) {
-        console.error("Sorry, i can`t parse to JSON indexDATA file. \n" + error);
+        throw new Error (`Sorry, i can not parse to JSON indexDATA file. \n ${error}`)
```
Comment (fedotov) [inline src/server/services/helpers/setFileIndex.js:25]: Same as above
